### PR TITLE
Dependency bumps before the next release

### PR DESCRIPTION
**Pull request by Verdent**
Dependency bumps before the next release

### CODE REVIEW

**Review by m0mus — APPROVED** [review]

File: yasson-tck/pom.xml [hunk 1]
```diff
@@ -11,11 +11,11 @@
...
         <jsonb.tck.version>3.0.0</jsonb.tck.version>
-        <yasson.version>3.0.0-SNAPSHOT</yasson.version>
-        <jakarta.json.bind.version>3.0.0</jakarta.json.bind.version>
-        <jakarta.json.version>2.1.0</jakarta.json.version>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+        <yasson.version>3.0.4-SNAPSHOT</yasson.version>
+        <jakarta.json.bind.version>3.0.1</jakarta.json.bind.version>
+        <jakarta.json.version>2.1.3</jakarta.json.version>
+        <maven.compiler.source>17</maven.compiler.source>
```
Comment (pzygielo) [inline yasson-tck/pom.xml:17]: This seems to be causing https://github.com/eclipse-ee4j/glassfish/pull/25017#issuecomment-2209585490

Comment (Verdent) [reply yasson-tck/pom.xml:17]: I will prepare a fix for this and spin staging re-release. Thanks :-)

Comment (pzygielo) [reply yasson-tck/pom.xml:17]: Just to clarify - unfortunate I commented here, as it should rather be about change in main `pom.xml` in first place probably.
(I missed the change there for _large diffs are not displayed by default_, and picked the next visible one.)
:grin:

Comment (Verdent) [reply yasson-tck/pom.xml:17]: Do not worry, I know :-) It will be changed there.

**Pull request merged**